src/main/java/nl/tudelft/semgroup4/StartScreenStateController.java

```diff
@@ -0,0 +1,203 @@
+     *            - {@link StateBasedGame} - The StateBasedGame to pass to the Game object.
+     * @return {@link Game} - A game object.
+     */
+    protected Game createSingleplayerGame(GameContainer container, StateBasedGame mainApp) {
+        final Player player =
+                new Player(new ResourcesWrapper(), container.getWidth() / 2,
+                        container.getHeight() - resources.getPlayerImageStill().getHeight()
+                                - 5 * resources.getWallImage().getHeight(), true);
+        final Game singleplayerGame =
+                new SingleplayerGame(mainApp, container.getWidth(), container.getHeight(),
+                        resources, player);
+        singleplayerGame.getCurLevel().toAdd(player.getWeapon());
+        PlayerEventHandler player1Handler = new PlayerEventHandler(player);
+        Settings.getPlayer1Input().addObserver(player1Handler);
+
+        return singleplayerGame;
+    }
+
+    /**
+     * Creates a new {@link MultiplayerGame}.
+     * 
+     * @param container
+     *            {@link GameContainer} - The state's Gamecontainer.
+     * @param mainApp
+     *            - {@link StateBasedGame} - The StateBasedGame to pass to the Game object.
+     * @return {@link Game} - A game object.
+     */
+    protected Game createMultiplayerGame(GameContainer container, StateBasedGame mainApp) {
+        Player firstPlayer =
+                new Player(new ResourcesWrapper(), container.getWidth() / 2,
+                        container.getHeight() - resources.getPlayerImageStill().getHeight()
+                                - 5 * resources.getWallImage().getHeight(), true);
+        PlayerEventHandler player1Handler = new PlayerEventHandler(firstPlayer);
+        Settings.getPlayer1Input().addObserver(player1Handler);
+
+        Player secondPlayer =
+                new Player(new ResourcesWrapper(), container.getWidth() / 2 + 100,
+                        container.getHeight() - resources.getPlayerImageStill().getHeight()
+                                - 5 * resources.getWallImage().getHeight(), false);
+        PlayerEventHandler player2Handler = new PlayerEventHandler(secondPlayer);
+        Settings.getPlayer2Input().addObserver(player2Handler);
+
+        final Game multiplayerGame =
+                new MultiplayerGame(mainApp, container.getWidth(), container.getHeight(),
+                        new ResourcesWrapper(), firstPlayer, secondPlayer);
+
+        multiplayerGame.getCurLevel().toAdd(firstPlayer.getWeapon());
+        multiplayerGame.getCurLevel().toAdd(firstPlayer.getWeapon());
+
+        return multiplayerGame;
+    }
+
+    /**
+     * Creates the correct {@link MouseOverArea} for the 1 Player button.
+     * 
+     * @param container
+     *            {@link GameContainer} - the container to draw in.
+     * @return {@link MouseOverArea} - The MouseOverArea for this button.
+     */
+    protected MouseOverArea createPlayer1Button(GameContainer container) {
```

Comment:
Doing that would require the state to know the location of the buttons. Now the controller simply knows everything and the state simply uses it.
The methods are there in the first place, because calling the MouseOverArea constructor causes a crash in Slick2d on Travis.

< interface, organisation, testing